.github/workflows/_61_upload_engine_shared_libs.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  workflow_call:
+
+permissions:
+  id-token: write
+  contents: read
+jobs:
+  upload:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout 🛒
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+
+      - name: Download Artifacts 📥
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a
+        with:
+          name: chainflip-backend-bin
+          path: chainflip-backend-bin
+
+      - name: Download M2 binaries 📥
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a
+        with:
+          name: chainflip-backend-bin-mac-m2
+          path: chainflip-backend-bin-mac-m2
+
+      - name: Set Version 🔦
+        id: version
+        shell: bash
+        run: |
+          chmod +x chainflip-backend-bin/chainflip-node
+          VERSION=$(./ci/scripts/extract_version.sh "$(./chainflip-backend-bin/chainflip-node --version)" full)
+          echo "Version: $VERSION"
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+          echo "url_friendly_version=$(echo $VERSION | tr . _)" >> $GITHUB_OUTPUT
+          echo "full_version=$VERSION" >> $GITHUB_OUTPUT
```

Comment:
Only `url_friendly_version` is used as far as I can tell, where are the others used? Also `version` and `full_version` are the same?